recipes/param_persist/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "param_persist" %}
+{% set version = "0.0.8" %}
+{% set sha256 = "fcf7180f544236683fb96b8fc80c752b8d34ec153abd4b54024c28cff1670bd5" %}
+{% set os_distribution = "linux-x86_64" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ os_distribution }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - sqlalchemy
+    - param
+
+test:
+  imports:
+    - pytest
+    - pytest-mock
+    - testfixtures
+
+about:
+  home: https://github.com/Aquaveo/param_persist
+  # Remember to specify the license variants for BSD, Apache, GPL, and LGPL.
+  # Prefer the short version, e.g: GPL-2.0 instead of GNU General Public License version 2.0
+  # See https://opensource.org/licenses/alphabetical
+  license: BSD-3-Clause
```

Comment:
license is listed as `BSD-2-Clause` in the upstream repo

Reply:
Fixed this